testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -420,6 +420,22 @@ impl SignerTest<SpawnedSigner> {
     }
 }
 
+fn verify_last_block_contains_tenure_change_tx(cause: TenureChangeCause) {
```

Comment:
Nit: can you change line 2806 to use this function? I guess you might need to refactor the function to not use `panic`/`assert_eq`. Perhaps the best change would be to have `last_block_contains_tenure_change -> bool`, and then also `verify_last_block_contains_tenure_change`.

Reply:
d8140e063